policies/attestation_task_bundle.rego

```diff
@@ -0,0 +1,36 @@
+package policies.attestation_task_bundle
+
+import data.lib
+
+allowed_bundles := ["quay.io/appstudio-utils:latest"]
```

Comment:
These seem to be the valid ones:

* quay.io/redhat-appstudio/build-templates-bundle
* quay.io/redhat-appstudio/hacbs-templates-bundle

Maybe just a prefix for now as there's no `latest` tag in those repos, only git commit tags which change very very frequently.

Reply:
Good idea. I'll update.